translate/filters/checks.py

```diff
@@ -2394,6 +2394,24 @@ def ftl_format(self, str1, str2):
         return True
 
 
+iosconfig = CheckerConfig(
+    varmatches=[("$(", ")"), ("%", "@")],
+)
+
+
+class IOSChecker(StandardChecker):
+
+    def __init__(self, **kwargs):
+        checkerconfig = kwargs.get("checkerconfig", None)
+
+        if checkerconfig is None:
+            checkerconfig = CheckerConfig()
+            kwargs["checkerconfig"] = checkerconfig
```

Comment:
I am not going to complain about this being copied again and again on every checker class.